webm3u2m3u.py

```diff
@@ -26,8 +26,8 @@ def quote_remover(string):
     protocol = quote_remover(opts.protocol[0])
     
     
-    if input.startswith('http') == True: #https://www.tutorialspoint.com/python/string_startswith.htm
-        data = urllib2.urlopen(input) #https://stackoverflow.com/a/1393367/11214013
+    if input.startswith('http'): # https://www.tutorialspoint.com/python/string_startswith.htm
+        data = requests.get(url=input).text
```

Comment:
Causes an issue at line 41. ```for line in data:```

No longer parses the input file properly. Returns a single character instead of entire line.